logic/LogicManager.java

```diff
@@ -56,12 +59,14 @@ public class LogicManager {
     private final ConceptManager conceptMgr;
     private final TypeInference typeInference;
     private final LogicCache logicCache;
+    private final Map<ResolvableConjunction, Set<Resolvable<?>>> compiled;
```

Comment:
`compiled` is not enough to provide the reader an indication of what the compilation is about... `compiledConjunctions` might be better since it implies it's "many" and to do with "conjunctions", so the name is a bit more standalone